packages/components/tag/src/tag.js

```diff
@@ -234,7 +114,7 @@ Tag.propTypes = {
   type: PropTypes.oneOf(['normal', 'warning']),
```

Comment:
What happened to the JS docs of the `<Tag>` component?

Reply:
oops 😅